src/utils/__tests__/conversion.ts

```diff
@@ -3236,7 +3236,7 @@ describe('claimToMeshClaim and meshClaimToClaim', () => {
       let value: Claim = {
         type: ClaimType.Jurisdiction,
         code: CountryCode.Cl,
-        scope: { type: ScopeType.Identity, value: 'someTickerDid' },
+        scope: { type: ScopeType.Identity, value: 'SOME_TICKERDid' },
```

Comment:
(nit) mixed case looks weird to me

```suggestion
        scope: { type: ScopeType.Identity, value: 'SOME_TICKER_DID' },
```

Reply:
My bad, I did a mass replace of `someTicker` to `SOME_TICKER`

Reply:
There's a few more spots like this still